src/chunk_item.rs

```diff
@@ -12,10 +12,12 @@ use zarrs::{
     storage::{MaybeBytes, ReadableWritableListableStorageTraits, StorageError, StoreKey},
 };
 
-use crate::utils::PyErrExt;
+use crate::{utils::PyErrExt, StoreConfig};
 
 pub(crate) type Raw<'a> = (
```

Comment:
For a later date but this makes me realize that we should probably turn this into a struct instead of just having to remember what exists at which position....no?

Reply:
#71